### DISCUSSION TITLE
articles/on-writing-first-draft-of-my-book/

### DISCUSSION THREAD

**Discussion by giscus** [piglei/piglei-giscus-comments#2] [General]
# articles/on-writing-first-draft-of-my-book/

piglei 的 blog

https://www.piglei.com/articles/on-writing-first-draft-of-my-book/

**Comment by yehexianyun** [comment]
从豆瓣简介顺藤摸瓜到这里来，作为一个python初学者，作者的经验和见解完全是老法师最真诚的忠告。

**Reply by piglei** [reply]
谢谢你顺藤摸瓜找到这特意给的好评！🥰
